infrastructure/cli/pkg/workspace/defaults.yaml

```diff
@@ -2,4 +2,4 @@ global:
   appImageTag: develop
   containerRegistry: ghcr.io/airyhq
   namespace: default
-  ngrokEnabled: true
+  ngrokEnabled: false
```

Comment:
So the default `airy create --provider=minikube` will not deploy Ngrok, correct?

Reply:
Yes it will. Because it returns it on the `GetHelmOverrides` :)